docs/user-guide/configuring.md

```diff
@@ -455,7 +455,7 @@ The complete configuration hierarchy, from highest precedence to lowest preceden
 3. Project-level configuration:
     1. `.eslintrc` file in same directory as linted file
     1. `package.json` file in same directory as linted file
-    1. Continue searching for `.eslintrc` and `package.json` files in ancestor directories (parent has highest precedence, then grandparent, etc.), up to and including the root directory or until a config with `"root": true` is found.
+    1. Continue searching for `.eslintrc` and `package.json` files in ancestor directories (parent has highest precedence, then grandparent, etc.), up to and including the root directory or until a config with `"root": true` is found. If there is a default configuration in `~/.eslintrc`, it will be ignored if the project has any local `.eslintrc` configurations.
```

Comment:
There is an exception to that rule, if the home folder is the same folder as the project folder the `.eslintrc` file will be applied (see #3846).